### DISCUSSION TITLE
建议将空调的室内温湿度传感器信息添加一份至 Sensor 组件

### DISCUSSION THREAD

**Discussion by SuLingGG** [XiaoMi/ha_xiaomi_home#260] [Ideas]
目前我拥有的两款空调：

- 小米3 匹新风 Pro (xiaomi.airc.rr0r00)
- 小米 1.5 匹新风 Pro (xiaomi.aircondition.m28)

空调自带的室内温湿度传感器信息是集成在以下两个空调控制器中的。

- climate.xiaomi_cn_xxxxx_rr0r00
- climate.xiaomi_cn_xxxxx_m28

这导致无法在后续场景中再次利用室内温湿度的传感器数值，比如无法将温湿度加入传感器和统计图表卡片中。
建议将空调温湿度传感器的测量值也复制一份增加至 Sensor 组件中，现有空调控制器中显示的数值无需删除。

**Comment by SusanPhevos** [comment]
请参考 #1009
